### PR TITLE
refactor: tefi-server thread api moved to tefi-electron-app

### PR DESCRIPTION
**Pull request by abrehmaaan**

### CODE REVIEW

File: data/useThreadById.ts [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-  if (!id) return null;
-  return `${CLUB_SERVER_ROOT}/dagora/thread/${id}?isTestnet=${process.env.NEXT_PUBLIC_IS_TESTNET ? true : false}`;
+const getKey = async (id: string) => {
+  try {
+    if (!id) return null;
+    const parseId = parseInt(id);
+    const result = await queryClientForThreadById(parseId, process.env.NEXT_PUBLIC_IS_TESTNET ? true : false);
+    return result;
+  } catch (err) {
+    return null;
```
Comment (Fahad-Mahmood) [inline data/useThreadById.ts]: key should return unique url value , here in this case it should return id only. 
fetcher function will be replaced by queryClientForThreadById, we can now check for testnet inside that function. 

Please complete the learning task for react-swr as well to understand this better.

File: constants.ts [hunk 2]
```diff
@@ -6,6 +6,11 @@ export const TERRA_MAIN_NET = { URL: 'https://lcd.contco.dev', chainID: 'columbu
 export const TERRA_OBSERVER_URL = 'wss://observer.terra.dev';
 export const TERRA_SWAP_GRAPHQL_URL = TEFI_API + 'https://terraswap-graph.terra.dev/graphql';
 export const CLUB_SERVER_ROOT = 'https://tef-club-server.herokuapp.com';
+export const FCD_URL = 'https://fcd.terra.dev';
+export const LCD_URL = 'https://lcd.terra.dev';
+export const TESTNET_LCD_URL = 'https://pisco-lcd.terra.dev';
+export const DGORA_TESTNET_CONTRACT_ADDRESS = 'terra18luvq9wv88lk0h5xvx2eatnssqew2z36g7f2q0zhk4337q4g7wnqzwhe9w';
+export const MICRO = 1000000;
```
Comment (Fahad-Mahmood) [inline constants.ts]: MICRO, LCD, FCD is already added to the terra-utilities package, we can re-use them from the package.

Comment (abrehmaaan) [reply constants.ts]: Done

File: data/useThreadById.ts (hunk 1 above)
Comment (abrehmaaan) [reply data/useThreadById.ts]: Working on it

**Comment by abrehmaaan** [comment]
@Fahad-Mahmood Please check now

**Review by Fahad-Mahmood — APPROVED** [review]
Looks Great 🔥

**Pull request merged**